### ISSUE TITLE
Problem with display scaling

### ISSUE THREAD

**Issue by Emme** [aza547/wow-recorder#123]
**Describe the bug**
When using windows display scaling, the app can't find a matching output resolution:

> [error] [OBS] ERROR! Matching resolution not found for Video Output
> [error] Error attempting to match 3841.25x2161.25 with 1920x1080,1280x720,1920x1200,1200x1920,3840x2160 for Base

**Expected behavior**
The output resolution should be set to the closest matching one

**Additional context**
The log entry for the relevant monitor:
```
  {
    id: 2841568472,
    bounds: { x: 0, y: 0, width: 2195, height: 1235 },
    workArea: { x: 0, y: 0, width: 2195, height: 1195 },
    accelerometerSupport: 'unknown',
    monochrome: false,
    colorDepth: 24,
    colorSpace: '{primaries:BT709, transfer:SRGB, matrix:RGB, range:FULL}',
    depthPerComponent: 8,
    size: { width: 2195, height: 1235 },
    displayFrequency: 60,
    workAreaSize: { width: 2195, height: 1195 },
    scaleFactor: 1.75,
    rotation: 0,
    internal: false,
    touchSupport: 'unknown'
  }
```
[WarcraftRecorder-2022-09-19.log](https://github.com/aza547/wow-recorder/files/9601497/WarcraftRecorder-2022-09-19.log)
[2022-09-19 19-29-11.txt](https://github.com/aza547/wow-recorder/files/9601498/2022-09-19.19-29-11.txt)

**Comment by aza547** [comment]
Interesting! Another user had trouble with the scaleFactor rounding. We made a fix in the last release to "snap" to a close resolution so long as the resolution was within 1 pixel. Here it's 1.25 pixels off so that snapping doesn't happen. 

Easy fix that we should probably make is to make the snapping happen for a wider range (bumping it to 2 would fix it - but that might not solve for everyone...).

**Comment by aza547** [comment]
@Emme I'd like to make a fix and share a rebuilt version of the app for you to test. Sound OK?

**Comment by aza547** [comment]
https://drive.google.com/file/d/1rIsXBHJ_-LHptbb_M2Kf8DLAIdWIKqlo/view?usp=sharing - let me know if that sorts it!

**Comment by Emme** [comment]
Yes, it's working now.

**Comment by aza547** [comment]
Great. I'll mark this closed and will be fixed in 2.7.1 when it releases.
